cmd/kpod/history.go

```diff
@@ -0,0 +1,327 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+	"text/template"
+	"time"
+
+	"os"
+
+	"github.com/Sirupsen/logrus"
+	is "github.com/containers/image/storage"
+	"github.com/containers/storage"
+	units "github.com/docker/go-units"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+// historyOutputParams stores info about each layer
+type historyOutputParams struct {
+	ID        string `json:"id"`
+	Created   string `json:"created"`
+	CreatedBy string `json:"createdby"`
+	Size      string `json:"size"`
```

Comment:
So size is an int64, but for the human flag, I am using the HumanSize function from docker/go-units and that outputs the size as a string. So I decided to convert it to string.